glib/src/boxed.rs

```diff
@@ -288,7 +288,7 @@ macro_rules! glib_boxed_wrapper {
             type Checker = $crate::value::GenericValueTypeOrNoneChecker<Self>;
```

Comment:
One line above there's another `Self`: `&'a $name $(<$($generic),+>)?` can become `&Self`.

Same in the other files

Reply:
Are you sure? I can't find that. By the way, you've commented on the wrong side, the side of upstream/master, which doesn't have the changes.

Reply:
Because GitHub's review interface sucks, I can't comment on the correct line.

See the end of this line: https://github.com/gtk-rs/gtk-rs-core/blob/ca43ffe5e344e06ababdbef533380ca3a2a6d252/glib/src/boxed.rs#L287

Reply:
I can't replace that with `Self`. I can't say `impl Display for Self`. What's `Self` in there?
I can only use `Self` after the target of the `impl` block has been specified

Reply:
Oh right, that was stupid. Sorry :)